src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountPageViewModel.cs

```diff
@@ -29,7 +29,7 @@ public class CreateAccountPageViewModel : ViewModel
 		{
 			var validationResult = await Form.Validate(ct);
 
-			if (validationResult.IsValid && Form.PasswordHasEightCharacters == true && Form.PasswordHasNumber == true && Form.PasswordHasUppercase == true)
+			if (validationResult.IsValid && Form.PasswordHasMinimumLength == true && Form.PasswordHasNumber == true && Form.PasswordHasUppercase == true)
```

Comment:
Since we are comparing bool and bool? in this condition, we have to compare with true to get a bool value to compare with validationResult.IsValid  (bool).